src/Data/Tree/DUAL.hs

```diff
@@ -2,7 +2,7 @@
 -----------------------------------------------------------------------------
 -- |
 -- Module      :  Data.Tree.DUAL
--- Copyright   :  (c) 2011-2012 Brent Yorgey
+-- Copyright   :  (c) 2011-2015 Brent Yorgey
```

Comment:
We should change the copyright from just me to "dual-tree team (see LICENSE)" and then list all contributors in the LICENSE file (as we do for other packages).